partcad/src/partcad/test/test.py

```
@@ -0,0 +1,57 @@
+#
+# Licensed under Apache License, Version 2.0.
+#
+
+from abc import ABC, abstractmethod
+import copy
+
+from .. import logging as pc_logging
+
+
+class Test(ABC):
+    def __init__(self, name) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `name`

(ANN001)